src/Library/demos/Video/main.blp

```diff
@@ -0,0 +1,30 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage page {
+  title: "Video";
+  description: _("A Widget that displays video with media controls");
+
+  Box {
+    orientation: vertical;
+    halign: center;
+
+    Video video {
+      width-request: 640;
+      height-request: 360;
```

Comment:
we should set this too the video we exact resolution of the video we will load

Reply:
On my 1080p Monitor the exact resolution renders the demo basically unusable, I currently have it set to half the resolution.